### ISSUE TITLE
Some exercise suggestions

### ISSUE THREAD

**Issue by cindRoberta** [exercism/vlang#2]
https://github.com/cindRoberta/V

https://github.com/serkonda7/v-coding-challenges

**Comment by loziniak** [comment]
[My approach](https://github.com/exercism/red/issues/31) was to randomly choose from `problem-specifications` repo, then arbitrarily assigning difficulty points. Good luck!

**Comment by iHiD** [comment]
Yeah, I'd highly recommend using the problem-specifications exercises like the other tracks :)

**Comment by keiravillekode** [comment]
We are relying on problem-specifications for exercises.
